docs/using_docker.md

```diff
@@ -0,0 +1,30 @@
+# Using Docker
```

Comment:
Explain that docker is used in production deployment. It's good to provide arguments when to use which type of method for running backend.